Author: akira yamada <akira@debian.org>
Description: Some files in the ruby source provide RDoc comments, but are not
included in a default rdoc run. This patch allows to generate the doc
for those files as well.
Debian specific patch
--- a/ext/.document
+++ b/ext/.document
@@ -1,6 +1,8 @@
 # Add files to this as they become documented
 
+bigdecimal/bigdecimal.c
 enumerator/enumerator.c
+gdbm/gdbm.c
 iconv/iconv.c
 nkf/lib/kconv.rb
 nkf/nkf.c
--- a/lib/cgi/.document
+++ b/lib/cgi/.document
@@ -1,2 +1,2 @@
+session
 session.rb
-
